internal/k8s/pod.go

```diff
@@ -105,6 +105,20 @@ func NodeIDFromPod(pod *v1.Pod) NodeID {
 	return NodeID(pod.Spec.NodeName)
 }
 
+func RestartsFromPod(pod *v1.Pod) int {
+	var restarts int32 = 0
+	for _, cStatus := range pod.Status.ContainerStatuses {
+		if cStatus.Name == "tilt-synclet" { // this should be a constant but there's nowhere good to put it :(
```

Comment:
any thoughts on where to put this? ought to go in `sidecar.go` but then i get circular dependencies with `sidecar` and `k8s` packages :(

Reply:
for better or worse, I've been trying to avoid adding ways in which `k8s` is tilt-specific, which would lead me to either:
1. make this more generic, so that tilt can do what it needs (e.g., `func RestartsFromPod(pod *v1.Pod, []string containersToIgnore)`, `func RestartsFromContainer(pod *v1.Pod, ContainerName)`, `func RestartsFromPod(pod *v1.Pod) (restartCountByContainerName map[ContainerName]int)`
2. just move the whole function out of `k8s`

I think I'd lean towards #2, since all the signatures in #1 seem kinda ugly. There's not really so much code here that it'd be awful to put it in the engine package.

Reply:
kk. put in engine/pod.go -- not suuuure that's the right place for it?